mongo/readpref/options.go

```diff
@@ -58,3 +58,13 @@ func WithTagSets(tagSets ...tag.Set) Option {
 		return nil
 	}
 }
+
+// WithHedgeEnabled specifies whether or not hedged reads should be enabled in the server. This feature requires MongoDB
```

Comment:
nit: add what the default value for this is.

Reply:
Done. This is a little tricky because there isn't a single default. If the user never calls `WithHedgeEnabled`, we send nothing to the server and it uses its defaults. However, those defaults can be changed via server startup parameters.